app/views/layouts/application.html.haml

```diff
@@ -15,6 +15,8 @@
       = favicon
     - else
       = favicon_link_tag '/favicon.ico'
+    = yield :head_meta
+    = yield :head_js
```

Comment:
what are these lines for? I can't tell if they are being used.

Reply:
they survived from action_cable ill remove